languages/go/exercises/.docs/README.md

```diff
@@ -0,0 +1,15 @@
+# Go exercise documents
```

Comment:
The location for this file should change to `exercises/shared/.docs/README.md`. See #54.